python/whylogs/api/logger/__init__.py

```diff
@@ -31,12 +31,14 @@
     ModelPerformanceMetrics,
 )
 from whylogs.core.stubs import pd
+from whylogs.core.utils import deprecated_argument
 
 diagnostic_logger = logging.getLogger(__name__)
 
 Loggable = Union["pd.DataFrame", List[Dict[str, Any]]]
 
 
+@deprecated_argument("debug_event")
```

Comment:
consider adding a message such as, "this parameter is no longer supported and will be removed in the next major version of whylogs."

Reply:
There's a similar default message, but I like your wording better.